server/db.js

```diff
@@ -1,4 +1,5 @@
 const mongoose = require("mongoose");
+require('dotenv').config({ path: "./sample.env" });
```

Comment:
I believe this is already included in another file and wouldn't be required here.

Reply:
Also, the `sample.env` is an example env so for future collaborators in a project can know what variables are needed in their environment to spin up the application.

The idea is for you to copy the `sample.env` and make a file called `.env`

Reply:
noted with thanks

Reply:
Although I had an issue with the server refusing to read the MONGO_URI env variable. The server could not read the variable until I specified the path as you saw. Notwithstanding, I will located the file where the `dotenv` package was required and see if I can fix the issue and discard my line. Thanks so much.